superset/assets/javascripts/explore/stores/controls.jsx

```diff
@@ -1086,6 +1087,14 @@ export const controls = {
     description: t('Whether to display the legend (toggles)'),
   },
 
+  show_annotation: {
```

Comment:
The term annotation is already used for other purposes, can we just call this `show_values`?

Reply:
noted